bluesky_queueserver/manager/tests/test_profile_ops.py

```diff
@@ -1031,7 +1031,11 @@ def test_devices_from_nspace():
     nspace = load_profile_collection(pc_path)
     devices = devices_from_nspace(nspace)
     for name, device in devices.items():
-        assert isinstance(device, ophyd.Device), f"The object '{device}' is not an Ophyd device"
+        assert isinstance(device, ophyd.ophydobj.OphydObject), f"The object '{device}' is not an Ophyd Object"
```

Comment:
As long as we are revisiting this, I think it's a good time to discard the `isinstance` check entirely and do a duck-type check instead. Could we vendor, for example, the `Readable` class proposed in https://github.com/bluesky/bluesky/pull/1446?

Reply:
It looks like Readable class will work for picking `ophyd.Signal` and `ophyd.Device` subclasses. There is also an advantage that there will be no `import ophyd` in the Queue Server code.

Reply:
Works for me.

Reply:
@danielballan I will write the issue so that we will not forget